src/settings/index.js

```diff
@@ -13,80 +19,105 @@ import PaymentSettings from './PaymentSettings';
 import CommentRequiredSettings from './CommentRequiredSettings';
 import RefundReasonsSettings from './RefundReasonsSettings';
 
-const general = [
-  {
-    route: 'perms',
-    label: 'Permission sets',
-    component: PermissionSets,
-    perm: 'ui-users.editpermsets',
-  },
-  {
-    route: 'groups',
-    label: 'Patron groups',
-    component: PatronGroupsSettings,
-    perm: 'ui-users.settings.usergroups',
-  },
-  {
-    route: 'addresstypes',
-    label: 'Address Types',
-    component: AddressTypesSettings,
-    perm: 'ui-users.settings.addresstypes',
-  },
-  {
-    route: 'profilepictures',
-    label: 'Profile pictures',
-    component: ProfilePictureSettings,
-  },
-];
+class UsersSettings extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+  };
+
+  // eslint-disable-next-line react/sort-comp
+  render() {
+    return (
+      <Settings
+        {...this.props}
+        sections={this.getSections()}
+        paneTitle={<FormattedMessage id="ui-users.settings.label" />}
+      />
+    );
+  }
+
+  getSections() {
+    return [
+      {
+        label: <FormattedMessage id="ui-users.settings.general" />,
+        pages: _.sortBy(this.getGeneral(), ['label']),
```

Comment:
Not sure the sorting will continue to work as `label` prop becomes `FormattedMessage` component now instead of string.